diff --git a/node_modules/victory-native/src/cartesian/hooks/useBarPath.ts b/node_modules/victory-native/src/cartesian/hooks/useBarPath.ts
index 1958f44..a2218ba 100644
--- a/node_modules/victory-native/src/cartesian/hooks/useBarPath.ts
+++ b/node_modules/victory-native/src/cartesian/hooks/useBarPath.ts
@@ -47,7 +47,7 @@ export const useBarPath = (
         );
         path.addRRect(nonUniformRoundedRect);
       } else {
-        path.addRect(Skia.XYWHRect(rect.x, rect.y, rect.width, rect.height));
+        path.addPath(Skia.Path.Rect(Skia.XYWHRect(rect.x, rect.y, rect.width, rect.height)));
       }
     });
 
